subsys/fs/Kconfig

```diff
@@ -33,6 +33,14 @@ config SYS_LOG_FS_LEVEL
 
 if FILE_SYSTEM
 
+config APP_LINK_WITH_FS
```

Comment:
I do not see this being used

Reply:
https://github.com/zephyrproject-rtos/zephyr/blob/master/cmake/app/boilerplate.cmake#L318